MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs

```diff
@@ -23,10 +23,10 @@ internal class RigidBody : IRigidBody
         private Queue<Action<Rigidbody>> _updateActions = new Queue<Action<Rigidbody>>();
 
         /// <inheritdoc />
-        public MWVector3 Velocity { get; set; }
+        public MWVector3 Velocity { get; set; } = new MWVector3();
```

Comment:
Can `set;` be removed? Do you need to have the property setter?

Reply:
Yes.  We set the velocity from the patching utils method.

Reply:
Is that in a different file? I only see Velocity.FromUnityVector3 in the Update method.

---
In reply to: [307493966](https://github.com/microsoft/mixed-reality-extension-unity/pull/95#discussion_r307493966) [](ancestors = 307493966)

Reply:
BTW, not a blocker, just was wondering. PR approved.

---
In reply to: [307547602](https://github.com/microsoft/mixed-reality-extension-unity/pull/95#discussion_r307547602) [](ancestors = 307547602,307493966)

Reply:
Yes.  It is in the PatchingUtils.cs file for generating the rigid body patches.